src/z3c/dependencychecker/dependencychecker.py

```diff
@@ -84,6 +84,57 @@
 ['\"]        # Single or double quote.
 """, re.VERBOSE)
 
+FTI_BEHAVIOR_PATTERN = re.compile(r"""
+\s           # Whitespace.
+<element     #
```

Comment:
If possible, I'd just do it with a regex.

Reason: this way it is the same kind of code as the other checks.

A reason to use an xml parser: a *specific* example in a public plone package for which the regex would fail and for which a small regex fix is not possible. The string `maybe` isn't a good enough example :-)